service/src/test/java/bio/terra/landingzone/stairway/flight/StepsDefinitionFactoryTypeTest.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.landingzone.stairway.flight;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@Tag("unit")
+class StepsDefinitionFactoryTypeTest {
+
+  @ParameterizedTest
+  @MethodSource("getCorrectValues")
+  void testCorrectValuesInDifferentCases(String value) {
+    assertNotNull(StepsDefinitionFactoryType.fromString(value));
```

Comment:
How does this work without StepsDefinitionFactoryType being imported?

Reply:
In this case the code is in a different package, so I certainly would expect an import in this test file. Or is some magic going on to link test files to their associated source file?

Reply:
Similar situation, but a little bit different. Name of the packages for original class and its test class are the same because we mirror package names. So, I think jvm will be confused if we import package with the same name. This is true not only for this particular class.